### DISCUSSION TITLE
Some questions about running python api/index.py

### DISCUSSION THREAD

**Discussion by KaneZhang1211** [jyjblrd/Mocap-Drones#47] [Q&A]
I have installed all the independences.But some errors occurred.
![5e6998bfe6724a54f4c5909e818257fc](https://github.com/jyjblrd/Low-Cost-Mocap/assets/139527361/853ca0ac-e35c-450b-953b-194342399e62)

**Comment by cultist123** [comment]
Connect your microcontroller and burn the program. Enter the command ls /dev/cu.* in the terminal, and modify the code in index.py.

**Comment by jyjblrd** [comment]
As @cultist123 said, this means that there is no esp32 plugged in to send the commands. You need to change the line of code to be the correct serial port for your esp32. If you dont have an esp32 yet, you can just comment out the code that sends data to the serial port.

**Comment by AliHusseinAlmoussawi** [comment]
@KaneZrg Did you solve this problem? If yes, please if you can help me.
